apps/game/src/id.rs

```diff
@@ -0,0 +1,69 @@
+use map_model::{AreaID, BuildingID, IntersectionID, LaneID, ParkingLotID, RoadID, TransitStopID};
+use sim::{AgentID, CarID, PedestrianID};
+
+#[derive(Clone, Hash, PartialEq, Eq, Debug, PartialOrd, Ord)]
+pub enum ID {
```

Comment:
`game` duplicates the ID enum and adds the cases for agents. I also thought about making `map_gui::ID` generic and have a `Foreign` case, but that would be way more complicated